### PR TITLE
Initial support for MP OpenAPI 1.1.1

### PR DESCRIPTION
**Pull request by arturdzm**
OpenLiberty Pull Requester,

ATTENTION, READ THIS: Updated 4/11/2018 - Read and understand this completely,
then delete this entire template. If a reviewer or merger sees this template,
they should fail the review or merge.

If this code change is fixing a user-visible bug in previously released code, it MUST
have an associated issue mentioned in the PR text or description. That Issue also
MUST be labelled “release bug”

This directs automation to scrape this fix for inclusion in the next release's
list of bugs fixed.

If this code is NOT for fixing a released bug, for example new function, fixing
a bug in unreleased function, or other improvements that do not affect the
user-space, no label is needed. An issue could or could not be used based
on the developer’s discretion, but do still delete this text block.

For full details, please see this wiki page:

https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions

### CODE REVIEW

**Comment by arturdzm** [comment]
#build

**Comment by arturdzm** [comment]
#build

**Comment by bsbyrd1** [comment]
It doesn't look like ID Review is required. If a review is needed, please let us know. Thank you!

**Comment by arturdzm** [comment]
@bsbyrd1 Yes, there are no new messages

File: dev/com.ibm.ws.microprofile.openapi.1.1.model/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.openapi.1.1.model/bnd.bnd]: Are the copyright dates correct on all these new files?

Comment (arturdzm) [reply dev/com.ibm.ws.microprofile.openapi.1.1.model/bnd.bnd]: Good point, I will update it to 2019 for 1.1 project

**Comment by arturdzm** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_EeuLQDxqEem5aoSbj2kEbg&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

**Comment by wraschke** [comment]
For an explanation of why I reverted this, please see https://ibm-cloud.slack.com/archives/C30NGUQ9E/p1551921710258400

**Pull request merged**